tests/functional/adapter/conftest.py

```diff
@@ -1,4 +1,47 @@
 import pytest
+import os
+
+from dbt.exceptions import DbtDatabaseError
```

Comment:
Due to https://github.com/dbt-labs/dbt-adapters/discussions/87:
```suggestion
from dbt_common.exceptions import DbtDatabaseError
```